cranelift/Cargo.toml

```diff
@@ -19,7 +19,6 @@ path = "src/clif-util.rs"
 [[test]]
 name = "filetests"
 path = "tests/filetests.rs"
-harness = false
```

Comment:
If you're feeling particularly intrepid, this entire block can be deleted now.

Reply:
:fire: